docs/docs/integrations/embedding-stores/couchbase.md

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 7
+---
+
+# Couchbase
+
+https://www.trychroma.com/
```

Comment:
wrong URL

Reply:
Can't believe I've missed that :D 
Thank you!